### ISSUE TITLE
Protobuf error executing gym-ignition from gym-ignition:latest image

### ISSUE THREAD

**Issue by diegoferigo** [robotology/gym-ignition#76]
Executing the example `launch_cartpole.py` from the minimal image `diegoferigo/gym-ignition:latest` (https://github.com/robotology/gym-ignition/pull/77) fails with the following error:

```
root@3fd84b6f0e1f:/# python -O /github/examples/python/launch_cartpole.py 
pybullet build time: Oct 11 2019 21:23:09
INFO: Making new env: CartPoleDiscrete-Gazebo-v0
[Msg] Loading SDF string. File path not available.
[libprotobuf ERROR google/protobuf/descriptor_database.cc:57] File already exists in database: peer_info.proto
[libprotobuf FATAL google/protobuf/descriptor.cc:1164] CHECK failed: generated_database_->Add(encoded_file_descriptor, size): 
terminate called after throwing an instance of 'google::protobuf::FatalException'
  what():  CHECK failed: generated_database_->Add(encoded_file_descriptor, size): 
Aborted (core dumped)
```

The image follows the User Setup described in the README, that installs gym-ignition from the bdist distribution. I suspect that the linking ign-gazebo statically in the bindings shared library during the wheel creation might be the responsible. @traversaro

**Comment by diegoferigo** [comment]
As a reference, these are the shared libraries of the bindings file installed in Developer mode:

```
gym-ignition on  devel [⇣$] via 🐍 v3.6.8(ve) 
✦ ❯ ldd /rl/local/lib/python3.6/site-packages/_gympp_bindings.so | grep ign
        libignition-gazebo2.so.2 => /rl/local/lib/libignition-gazebo2.so.2 (0x00007fdf1de25000)
        libignition-fuel_tools3.so.3 => /rl/local/lib/libignition-fuel_tools3.so.3 (0x00007fdf1dbae000)
        libignition-gui2.so.2 => /rl/local/lib/libignition-gui2.so.2 (0x00007fdf1d96a000)
        libignition-common3-profiler.so.3 => /rl/local/lib/libignition-common3-profiler.so.3 (0x00007fdf1d75c000)
        libignition-common3-events.so.3 => /rl/local/lib/libignition-common3-events.so.3 (0x00007fdf1d557000)
        libignition-common3-av.so.3 => /rl/local/lib/libignition-common3-av.so.3 (0x00007fdf1d34b000)
        libignition-common3-graphics.so.3 => /rl/local/lib/libignition-common3-graphics.so.3 (0x00007fdf1ade1000)
        libignition-common3.so.3 => /rl/local/lib/libignition-common3.so.3 (0x00007fdf1ab94000)
        libignition-plugin1-loader.so.1 => /rl/local/lib/libignition-plugin1-loader.so.1 (0x00007fdf1a980000)
        libignition-plugin1.so.1 => /rl/local/lib/libignition-plugin1.so.1 (0x00007fdf1a775000)
        libignition-transport7-log.so.7 => /rl/local/lib/libignition-transport7-log.so.7 (0x00007fdf1a50f000)
        libignition-transport7.so.7 => /rl/local/lib/libignition-transport7.so.7 (0x00007fdf17fcc000)
        libignition-msgs4.so.4 => /rl/local/lib/libignition-msgs4.so.4 (0x00007fdf178fa000)
        libignition-math6.so.6 => /rl/local/lib/libignition-math6.so.6 (0x00007fdf1735c000)
```

And these in User mode:

```
root@4444c60b54e0:/# ldd /venv/lib/python3.6/site-packages/_gympp_bindings.so | grep ign
        libignition-common3.so.3 => /usr/lib/x86_64-linux-gnu/libignition-common3.so.3 (0x00007fe333b57000)
        libignition-fuel_tools3.so.3 => /usr/lib/x86_64-linux-gnu/libignition-fuel_tools3.so.3 (0x00007fe3338e9000)
        libignition-common3-events.so.3 => /usr/lib/x86_64-linux-gnu/libignition-common3-events.so.3 (0x00007fe3336e4000)
        libignition-transport7.so.7 => /usr/lib/x86_64-linux-gnu/libignition-transport7.so.7 (0x00007fe332afa000)
        libignition-msgs4.so.4 => /usr/lib/x86_64-linux-gnu/libignition-msgs4.so.4 (0x00007fe33260c000)
        libignition-math6.so.6 => /usr/lib/x86_64-linux-gnu/libignition-math6.so.6 (0x00007fe332061000)
        libignition-plugin1-loader.so.1 => /usr/lib/x86_64-linux-gnu/libignition-plugin1-loader.so.1 (0x00007fe3319f5000)
        libignition-plugin1.so.1 => /usr/lib/x86_64-linux-gnu/libignition-plugin1.so.1 (0x00007fe3317ea000)
```

Both have also:
```
libprotobuf.so.10 => /usr/lib/x86_64-linux-gnu/libprotobuf.so.10 (0x00007fc8c3fda000)
```

**Comment by diegoferigo** [comment]
Old issue https://github.com/google-code-export/protobuf/issues/370

**Comment by diegoferigo** [comment]
I opened upstream [ignitionrobotics/ign-gazebo#437](https://bitbucket.org/ignitionrobotics/ign-gazebo/pull-requests/437) to add the missing components we need. If it get accepted, we can switch back to use the official `ign-gazebo` package, removing the need of the workaround to compile my fork statically.

**Comment by diegoferigo** [comment]
For the sake of completeness, since the physics system is a plugin, another option to static compilation is vendoring in this repo a modified version of the plugin, and edit the world files to load our own. Also this solution would remove the need of the fork (I don't know why I never considered this option before). Though, merging the changes upstream is definitely the best approach.

**Comment by diegoferigo** [comment]
Fixed by #82
